src/OpenRealEstate.Core/Address.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
nitpic: This needed?

Reply:
system is for exception. i dont know why its coming up as a change it was always there

Reply:
Ok - so where's `Exception` in this class?

Reply:
line 183

Reply:
damn i sometimes hate PR view where it shows such a small section and you _think_ that's the entire file.